plugin/trino-hive/src/main/java/io/trino/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -90,6 +90,7 @@
 import static com.google.common.base.Preconditions.checkState;
```

Comment:
commit message is too long.

Reply:
We follow https://chris.beams.io/posts/git-commit/

Reply:
Okay. I will reword the commit message.